README.md

```diff
@@ -297,7 +398,7 @@ gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV
 --DLPMethod=REID 
 --keyRange=1024 
 --queryPath=gs://${REID_QUERY_BUCKET}/reid_query.sql 
---serviceAccount=$SERVICE_ACCOUNT_EMAIL"
+--serviceAccount=${SERVICE_ACCOUNT_EMAIL}"
 ```
 
 This command will trigger a batch re-identification Dataflow pipeline that will process all the records from the query 
```

Comment:
Please check if "Pipeline parameters" table is missing any critical parameters. I couldn't find filePattern.